_extras/guide.md

```diff
@@ -302,7 +305,8 @@ web-hosted private repositories.
 [git-parable]: http://tom.preston-werner.com/2009/05/19/the-git-parable.html
 [github]: https://github.com/
 [github-gui]: https://git-scm.com/downloads/guis
-[github-line-endings]: https://help.github.com/articles/dealing-with-line-endings/#platform-all
+[github-line-endings]: https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings#platform-all
```

Comment:
```suggestion
[github-line-endings]: https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings
```

Reply:
Thanks. I've made this change now, ready for review again.